dev/Pager/PrototypePager/PrototypePager.cs

```diff
@@ -24,6 +27,26 @@ public enum ButtonVisibilityMode { Auto, AlwaysVisible, HiddenOnEdge, None, }
 
         private Button FirstPageButton, PreviousPageButton, NextPageButton, LastPageButton;
         private ComboBox PagerComboBox;
+        private NumberBox PagerNumberBox;
+        private ItemsRepeater PagerNumberPanel;
+        private Rectangle NumberPanelCurrentPageIdentifier;
+
+        private static IconElement LeftEllipse = new SymbolIcon(Symbol.More);
+        private static IconElement RightEllipse = new SymbolIcon(Symbol.More);
+
+        private ObservableCollection<object> NumberPanelLeftMostState
```

Comment:
Interesting approach. I wouldn't have thought of doing it this way. Having an ObservableCollection with mixed contents like this is somewhat unusual. But I think it is a neat solution!